regression/Makefile

```diff
@@ -1,10 +1,12 @@
 # SPDX-License-Identifier: MPL-2.0
 
+VDSO_DIR := /root/dependency
+VDSO_LIB := $(VDSO_DIR)/vdso64.so
 MKFILE_PATH := $(abspath $(lastword $(MAKEFILE_LIST)))
 CUR_DIR := $(patsubst %/,%,$(dir $(MKFILE_PATH)))
 BUILD_DIR := $(CUR_DIR)/build
-VDSO_DIR := $(BUILD_DIR)/linux_vdso
 INITRAMFS := $(BUILD_DIR)/initramfs
+INIT_FILE := $(CUR_DIR)/benchmark/init
```

Comment:
This file name is inappropriate: the variable has very general name, `INIT_FILE`, but it only applies to benchmark.

Reply:
Also rename them to `BENCHMARK_ENTRYPOINT` and `benchmark_entrypoint.sh`.